doc/source/user_guide/hpc_slurm.rst

```diff
@@ -0,0 +1,611 @@
+Either because the virtual environment has been created in a directory which is not accessible
+from the nodes or because the virtual environment has been created from a Python executable
+which is not available to the compute nodes, hence the virtual environment is not activated.
+For example, you might be creating the virtual environment using Python 3.10, but only
+Python 3.8 is available from the compute nodes.
+
+You can test which Python executable the cluster is using by starting an interactive session in
+a compute node using:
+
+.. code-block:: console
+
+    user@machine:~$ srun --pty /bin/bash
+    user@compute_node_01:~$ compgen -c | grep python # List all commands starting with python
+
+.. the approach to solve this comes from:
+   https://stackoverflow.com/questions/64188693/problem-with-python-environment-and-slurm-srun-sbatch
+
+Many HPC infrastructure uses environment managers to load and unload software package
+using modules and environment variables. 
+Hence you might want to make sure that the correct module is loaded in your script.
+Two of the most common environment managers are
+`Modules documentation <modules_docs_>`_ and `Lmod documentation <lmod_docs_>`_.
+Check your cluster documentation to know which environment manager is using, and how to
+load Python with it. If you find any issue, you should contact your cluster administrator.
+
+If there is not a suitable Python version accessible from the compute nodes, you might need
+request to your HPC cluster administrator to have installed in all the compute
+nodes a suitable Python version.
+If this is not an option, visit :ref:`ref_ansys_provided_python`.
+
+.. _ref_ansys_provided_python:
+
+Using ANSYS provided Python installation
+----------------------------------------
+
+**For development purposes only**
+
+In certain HPC environments the possibility of installing a different Python version
+is limited for security reasons.
+
+In those cases, the Python distribution available within the ANSYS installation can be used.
+This Python distribution is a customized Python (CPython) version for ANSYS products use only, and
+its usage is **discouraged** except for very advanced users and user cases.
+
+This Python distribution is in:
+
+.. code-block:: text
+
+    /ansys_inc/v%MAPDL_VERSION%/commonfiles/CPython/3_10/linx64/Release/python
+
+whereas ``%MAPDL_VERSION%`` is the 3 digits ANSYS version. For instance for ANSYS 2024R2:
+
+.. code-block:: text
+
+    /ansys_inc/v242/commonfiles/CPython/3_10/linx64/Release/python
+
+
+From ANSYS 2024R1, the Python version included in the unified installer is CPython 3.10.
+Previous versions were including CPython 3.7 (``/commonfiles/CPython/3_7/linx64/Release/python``).
+
+Because ANSYS installation needs to be available to all the compute nodes to run simulations using them,
+this Python distribution is normally also available to the compute nodes.
+Hence, you can use it to create your own virtual environment.
```

Comment:
```suggestion
Because the Ansys installation must be available to all
the compute nodes to run simulations using them, this
Python distribution is normally also available to the compute
nodes. Hence, you can use it to create your own virtual
environment.
```